packages/react/package.json

```diff
@@ -24,6 +24,7 @@
     "dist/"
   ],
   "dependencies": {
+    "@mdi/js": "^7.2.96",
```

Comment:
should it be a dev dependency? as it stands it's appearing as a dependency in our built package, so everyone will get it added